nemo/collections/asr/models/classification_models.py

```diff
@@ -33,7 +33,6 @@
 __all__ = ['EncDecClassificationModel', 'MatchboxNet']
 
```

Comment:
You can also remove the `from nemo.utils.decorators import experimental` imports now

Reply:
thx, fixed